src/components/Pages/MyAssets/MintNft/MintNft.tsx

```diff
@@ -228,7 +227,12 @@ const MintNft = () => {
           <SInfoRow>
             <span>Mint Price</span>
```

Comment:
Maybe it's better to hide the whole block with that title if the mint price is not set?

Reply:
Currently showing "0 Unit" if there is no price

Reply:
If price is 0, then I can change the button text to "Mint for free"
otherwise show the price block as it is right now